.github/workflows/frontend.yml

```diff
@@ -24,9 +24,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: 19
+          node-version-file: frontend/package.json
```

Comment:
Automatically detect the `node` version that we set in our `frontend/package.json`, see https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

In the future, we only need to update the `node` version in `package.json`.

Reply:
oh nice!